sdk/identity/azure-identity/azure/identity/_credentials/default.py

```diff
@@ -42,3 +44,13 @@ def __init__(self, **kwargs):
             )
 
         super(DefaultAzureCredential, self).__init__(*credentials)
+
+    def get_token(self, *scopes, **kwargs):
+        try:
+            return super(DefaultAzureCredential, self).get_token(*scopes, **kwargs)
+        except ClientAuthenticationError as e:
+            raise ClientAuthenticationError(message="""
+{}\n\nPlease visit the Azure identity Python SDK docs at 
+https://aka.ms/python-sdk-identity#defaultazurecredential 
+to learn what options DefaultAzureCredential supports"""
+                .format(e.message))
```

Comment:
@chlowell  Shouldn't this also contain the error messages from the other credentials in the chain as well?

Reply:
It does, this catches `ChainedTokenCredential`'s exception, whose message has all that, and tacks `DefaultAzureCredential` help to the end.